### PR TITLE
update project to php5.3 and PSR2

### PR DESCRIPTION
**Pull request by trilopin**
There are structural changes which make PR Diff untraceable.
### Big changes
- Use namespace, -> php 5.3 is required
- all classes are in namespace Pt 
- **backwards compatibility broken**
### Coding standards PSR0, PSR1, PSR2:
- one class per file
- method and variable names in camelcase
- method and variable names must not prefix with underscore (visibility convention)
- in method arguments must have one space after comma
- .php_cs file with coding standards specs for php-cs-fixer tool
### Testing
- exceptions are tested with specific phpunit method setExpectedException instead try/catch 
### Continous integration
- appveyor setup, tests are running in windows environments
- travis setup: tests are running in php 5.3, php 5.4, php 5.5, php 5.6 and hhvm environments
### Delivery
- packagist setted in trilopin/ptmock. If you accept changes and want to mantain package i would transfer packagist ownership.
-  autoload via psr4 composer
### Documentation
- README renamed to README.md for markdown highlight
- appveyor and travis badges added to README

### CODE REVIEW

**Comment by trilopin** [comment]
- some properties still in underscore case changed to camelcase
- php-cs-fixer (dry-run mode) now runs from travis
- added scrutinizer for tracking code coverage and code analysis https://scrutinizer-ci.com/g/trilopin/pt_mock/

**Comment by gramos74** [comment]
I would like to change the license from MIT to Apache. So, as you already removed the header comments regarding the MIT license I would say that we can also change the LICENSE file and add comments in the source files related to the new Apache license.

File: README.md [hunk 1]
```diff
@@ -39,7 +41,7 @@ and we expect that it returns 'b'.
 To check that all expectations have been accomplished :
 
     $class_b->verify();     // for a instance of pt_mock
-    pt_mock::verify_all();  // for all mocks instantiated 
+    pt_mock::verify_all();  // for all mocks instantiated
```
Comment (gramos74) [inline README.md:49]: This method name should be changed. I would say that now is verifyAll, right ?

**Comment by trilopin** [comment]
it was accidental, when I was splitting files :(

we have this options for change license:
- I can update files and LICENSE in this pull request
- you can add pull request to my fork repo (and changes will be added to this pull request)
- do after merge

File: README.md (hunk 1 above)
Comment (trilopin) [inline README.md:49]: doc is outdated

Comment (trilopin) [inline README.md:49]: i have updated README in this PR

**Comment by trilopin** [comment]
i think this pull request is ready for merge

changes are big, @gramos74 please review

**Comment by trilopin** [comment]
all changes unified in one commit

**Pull request merged**